src/endpoints/seed/biographies.ts

```diff
@@ -0,0 +1,1332 @@
+    },
+    {
+      tenant: tenants['nwac'],
+      photo: 1,
+      name: 'Kendall Stever',
+      title: 'Treasurer',
+      start_date: '2017-10-01',
+      biography:
+        'Board member since 2017. Kendall has more than 30 years of professional accounting and finance experience in the high tech and life science industries.  A graduate of the University of Washington, he has held CFO and other executive level positions, started a consulting business and worked as an auditor with a Big 4 public accounting firm.  A native of Seattle, Kendall is a lifelong skier and backcountry adventurer.  Serving on the NWAC board is an ideal way to combine his love of the outdoors with his professional background while helping NWAC achieve its mission of educating the community and promoting safe travel in the backcountry.',
+    },
+    {
+      tenant: tenants['nwac'],
+      photo: 1,
+      name: 'Arun Jacob',
+      title: 'Vice President',
+      start_date: '2019-10-01',
+      biography:
+        'Arun moved to Seattle for a software internship in 1992, fell in love with the mountains and the water, and has been here ever since. He is excited to leverage his professional passion for data and analytics to help NWAC educate and equip the backcountry community with the information they need to be safe. In his free time, he gets outside as much as possible on a snowboard, a paddleboard, or a mountain bike.',
+    },
+    {
+      tenant: tenants['nwac'],
+      photo: 1,
+      name: 'Gavin Woody',
+      title: 'President',
+      start_date: '2017-12-01',
+      biography:
+        'As a passionate backcountry skier, father of two young children (who love to ski!), and an advocate for responsible recreating in the backcountry, I’m honored to have served on the NWAC Board since 2017. In addition to skiing, I love taking advantage of all the outdoor activities the PNW has to offer: ultrarunning, mountaineering. biking, hunting, and fishing. Originally from California, I served as a U.S. Army infantry captain and am a decorated combat veteran (Operation Iraqi Freedom). My professional background includes management consulting with McKinsey & Company and operations executive roles at Expedia, A Place For Mom, and Porch Group, where I currently lead our insurance agency. I hold a mechanical engineering degree, with honors, from the United States Military Academy at West Point and an MBA from the Stanford Graduate School of Business.',
+    },
+    {
+      tenant: tenants['snfac'],
+      photo: 1,
+      name: 'Louise Stumph',
+      title: 'Director',
+      start_date: null,
```

Comment:
⛏️  I noticed all of the `snfac` are `null`. Maybe make sure the have at least one in each tenant with and without the optional fields. Same goes for `title`. I think all `nwac` bios have a title. Maybe have one without a title

Reply:
This is simply a reflection of how the different centers are using the features today on their websites - so it was moreso pulling in their real data rather than trying to make sure some fields are or are not set.